src/backend/webpackBuild.js

```diff
@@ -4,6 +4,8 @@
 const webpack = require('webpack');
 const colors = require('colors');
 const webpackConfig = require('../../webpack.config.live.js').default;
+// TODO maybe call compiler.close(callback)
```

Comment:
take a look at the link there for upgrading webpack v4 -> v5. not sure that it matters, but maybe it would be good. lmk if you understand better than me